alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportIndexWorkflowAction.kt

```diff
@@ -604,7 +604,7 @@ class TransportIndexWorkflowAction @Inject constructor(
         val searchSource = SearchSourceBuilder().query(query)
         val searchRequest = SearchRequest(SCHEDULED_JOBS_INDEX).source(searchSource)
 
-        if (user != null && !isAdmin(user) && filterByEnabled) {
+        if (user != null && filterByEnabled) {
```

Comment:
Why are we removing the `!isAdmin(user)` piece? Admin users bypass the filter by check.